base/MaplGrid.F90

```diff
@@ -63,32 +65,26 @@ subroutine set(this, grid, unusable, rc)
     _UNUSED_DUMMY(unusable)
     ! At this point, this component must have a valid grid!
     !------------------------------------------------------
-    call ESMF_GridValidate(grid, RC=STATUS)
-    _VERIFY(STATUS)
+    call ESMF_GridValidate(grid, __RC__)
 
     this%ESMFGRID = GRID
 
 
 ! We keep these in the component's grid  for convenience
 !-------------------------------------------------------
 
-    call ESMF_GridGet(this%ESMFGRID, DistGrid=distgrid, dimCount=dimCount, RC=STATUS)
-    _VERIFY(STATUS)
-    call ESMF_DistGridGet(distGRID, deLayout=this%LAYOUT, RC=STATUS)
-    _VERIFY(STATUS)
+    call ESMF_GridGet(this%ESMFGRID, DistGrid=distgrid, dimCount=dimCount, __RC__)
+    call ESMF_DistGridGet(distGRID, deLayout=this%LAYOUT, __RC__)
 
-    call ESMF_VmGetCurrent(VM, rc=status)
-    _VERIFY(STATUS)
-    call ESMF_VmGet(VM, localPet=this%MYID, petCount=ndes, rc=status)
-    _VERIFY(STATUS)
+    call ESMF_VmGetCurrent(VM, __RC__)
+    call ESMF_VmGet(VM, localPet=this%MYID, petCount=ndes, __RC__)
 
 ! Vertical coordinate must exist and be THE THIRD DIMENSION
 ! ---------------------------------------------------------
 
     this%VERTDIM = 3
 
-    call MAPL_GridGet(this%ESMFGRID, localCellCountPerDim=COUNTS, RC=STATUS)
-    _VERIFY(STATUS)
+    call MAPL_GridGet(this%ESMFGRID, localCellCountPerDim=COUNTS, __RC__)
```

Comment:
Please do a global replace of `__RC__` with `_RC`.   Inside MAPL this is the preferred form.
```suggestion
    call MAPL_GridGet(this%ESMFGRID, localCellCountPerDim=COUNTS, _RC)
```